common/src/main/proto/TransportMessages.proto

```diff
@@ -162,8 +163,9 @@ message PbRegisterWorker {
   int32 fetchPort = 4;
   int32 replicatePort = 5;
   repeated PbDiskInfo disks = 6;
-  string requestId = 9;
   map<string, PbResourceConsumption> userResourceConsumption = 8;
+  string requestId = 9;
+  map<string, PbResourceConsumption> applicationResourceConsumption = 10;
```

Comment:
IMO the key of applicationResourceConsumption does not look so intuitive，
At first look,  I thought this must be the applicationId.